src/index.ts

```diff
@@ -154,6 +156,47 @@ function shouldSendFlowLogToSplunk(msg: string): boolean {
   return action === 'ACCEPT' && !rejectedProtocols.includes(protocol)
 }
 
+function getLogTimeStampRegex(logType: CloudWatchLogTypes): RegExp | undefined {
+  switch (logType) {
+    case CloudWatchLogTypes.app:
+      return /"@timestamp"\s*:\s*"(.*?)"/
+    case CloudWatchLogTypes.squid:
+      return /^(\d+\.\d{3})/
+    case CloudWatchLogTypes.syslog:
+      return /^(.+?)\s*ip/
```

Comment:
syslog won't always start ip, it has the date (possibly with spaces which is annoying) followed by the hostname (separated with a space.....again annoying).

The hostnames currently happen to be ip-x-x-x-x which is down to the current version of ubuntu we are running, this may well change, so I'd suggest revising this to look for the format rather than the thing following it.

An alternative......we _could_ configure syslog on ubuntu to explicitly use the iso8601 time 🤔

Reply:
That's a great point and I'll update it. I was thinking we might want some monitoring of the lambda logs to highlight when we're unable to extract the date and are falling back to defaults (like current time) otherwise it could fail silently due to issues with the regex and future changes like you've pointed out